src/components/skeleton/Header/Header.js

```diff
@@ -57,4 +60,9 @@ export default class Header extends Component {
     // (solution: put some sort of icon at the top of the navigation so its non-clickabel?:)
     setTimeout(() => this.setState({isNavOpening: false}), 1000);
   }
+
+  _reverseName(name) {
+    var nameInParts = name.split(' ');
+    return nameInParts[1] + ' ' + nameInParts[0];
+  }
```

Comment:
kijk er zijn hier verschillende dingen die je kan doen:

-> in de backend first en last name splitten met een migratie
-> zo naar de frontend sturen 
-> dan kan de frontend beslissen hoe de naam te tonen

of:

Op de hoofdpagina wordt gewoon de header getoond
maar op pagina's waar er meer functionaliteit is (vb spelers pagina) wordt die standaard header vervangen door een Toolbar
http://www.material-ui.com/#/components/toolbar

Zodat op spelers je niet de standaard header hebt maar keuzes om Sporta/Vttl competitie te zien, excel export etc


**Wat sowieso kan gebeuren is de naam in het profiel plaatsen
Of als tooltip van het user manneke (hmm das mss nog beter)**